dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -162,35 +177,18 @@ export const SignInGateSelector = ({
 			: undefined;
 
 	const personaliseComponentId = (
-		currentComponentId: string | undefined,
-	): string | undefined => {
-		if (!currentComponentId) return undefined;
+		currentComponentId?: string,
+		checkoutCompleteCookieData?: CheckoutCompleteCookieData,
+	): string => {
+		if (!currentComponentId) return '';
+		if (!personaliseSwitch) return currentComponentId;
```

Comment:
I believe this should pass all they cypress tests without needing to have tracking data state and keeps the useOnce hook that gets the test/variant separate from any personalisation as the component personsalation is independant to the AB test, whcih is always the Main test. 

I had to return "" to make TS happy. Need to triple check this edge case but I don't hink it should be a problem.